docs/blog/2023-09-06-distributed-key-generation.md

```diff
@@ -0,0 +1,154 @@
+2. **Verification Phase**: Each participant verifies the received shares they have received from other participants. If a participant identifies an incorrect or invalid share, they issue a public complaint against the sender. The accused party must then either correct the share or risk being deemed an unreliable dealer.
+3. **Finalization Phase:** During the final phase, all verified shares from every participant are aggregated. However, any participant found to be unreliable or invalid during the verification stage is excluded from this aggregation
+
+********************************************Metrics of the protocol:********************************************
+
+| Network Model | Fault tolerance | Adaptive adversary | Discrete log key | CommunicationComplexity | Comp. Complexity | Round Complexity |
+| --- | --- | --- | --- | --- | --- | --- |
+| Sync. | 1/2 | No | ✅ | $O(n\mathcal{B}(\kappa t))$ | $O(nt^3)$ | $O(R)$ |
+
+### Implementation: The Kyber Library 💻:
+
+Drand's implementation (DKG) for its distributed randomness beacon is executed using the Kyber Library, a specialized cryptographic library written in Golang, also commonly referred to as the Go programming language. The Kyber Library as forked by Drand provides a robust set of cryptographic primitives and utilities, enabling the highly secure and efficient deployment of Pedersen's DKG scheme within this framework.
+
+Check this out:
+
+[GitHub - drand/drand: 🎲 A Distributed Randomness Beacon Daemon - Go implementation](https://github.com/drand/drand)
+
+[GitHub - drand/kyber: Advanced crypto library for the Go language](https://github.com/drand/kyber)
+
+### Sinking or Swimming: A Deep Dive into Pedersen's Security 🤿 :
+
+There has been considerable debate and, in some quarters, a level of confusion regarding the security aspects of Pedersen's DKG. To provide a clearer understanding of the evolving perspectives on its security, it's beneficial to delve into a brief timeline that outlines key milestones in the research focused on this framework.
+
+😃 🏊🏾‍♂️ ****1991: Pedersen**** 
+
+In the early 90s, Pedersen initially introduced the DKG scheme. The security in the initial scheme lies in the security of the Joint-Feldman Protocol for verifiable secret sharing during the secret-sharing process
+
+🙄⚓**********2006: [Gennaro et al.](https://link.springer.com/article/10.1007/s00145-006-0347-3)** 
+
+Gennaro et al. took the Joint-Feldman protocol as described by Pedersen and they showed that a potential adversary can influence the contribution of the public key in a way that is not being uniformly chosen randomly. Therefore, the adversary can take the advantage to compromise two of the participants and gain information about the private key, which means it leaks information about the secret. As a result, this is a sinking phase for Pedersen, as it shows that it cannot be used as a secure DKG protocol.
```

Comment:
this reads a bit weirdly: 'a potential adversary can influence the contribution of the public key in a way that is not being uniformly chosen randomly'

perhaps 'a potential adversary can craft their contribution in a way that can influence the group public key such that it is not uniformly random'